bundle/EventListener/LegacyKernelListener.php

```diff
@@ -41,8 +44,22 @@ function() {
         );
     }
 
+    public function onConfigScopeChange( ScopeChangeEvent $event )
+    {
+        $this->resetKernelHandler();
+    }
+
     public function onConsoleCommand( ConsoleCommandEvent $event )
     {
+        $this->resetKernelHandler();
+
+        $eventDispatcher = $this->container->get( 'event_dispatcher' );
```

Comment:
Should I then just remove the containeraware and inject both the dispathcer and ezpublish_legacy.kernel_handler.cli?